v2/src/modules/search/ModuleSearch.vue

```diff
@@ -321,6 +321,28 @@ const tokensResult = computed(() => {
                     flag: marketData !== false
                 }
             })
+        // Check if hash is of type TOKEN_HASH and get token from tokenMarketInfo
+        if (search.hashType === HashType.TokenHash) {
+            const token = getEthereumTokenByContract(search.param)
+            if (token) {
+                return [
+                    {
+                        text: token.name,
+                        subtext: token.symbol ? token.symbol : token.contract,
+                        price: token.current_price ? formatUsdValue(new BN(token.current_price)).value : undefined,
+                        icon: token.image,
+                        contract: token.contract
+                    }
+                ]
+            }
```

Comment:
add a condition if (!token) and use getTokenInfoByContract contract to fecth for the DB, otherwise if you search nft contract nothing comes up